cmd/operator-sdk/generate/bundle/cmd.go

```diff
@@ -116,7 +119,6 @@ func (c *bundleCmd) addFlagsTo(fs *pflag.FlagSet) {
 	fs.BoolVar(&c.manifests, "manifests", false, "Generate bundle manifests")
 	fs.BoolVar(&c.metadata, "metadata", false, "Generate bundle metadata and Dockerfile")
 
-	fs.StringVar(&c.operatorName, "operator-name", "", "Name of the bundle's operator")
```

Comment:
I do wonder whether it is a good idea to remove this flag. On one hand, `projectName` should be used, it should always exist in PROJECT, and typically only one operator bundle exists in an operator project. On the other, you can technically have more than one operator bundle per repo, perhaps named differently to denote configuration for different cloud providers.

Reply:
IMO remove the flag is better because keep it simple 👍 (we can add things after 1.0.0 but not easily remove :-) )

Reply:
I feel like there's a one-to-one mapping between bundle and PROJECT file as well. For example, the PROJECT file also contains the APIs that are part of the operator.